src/mono_hash_map.rs

```diff
@@ -18,8 +18,11 @@ use crate::AllocMap;
 pub struct MonoHashMap<K: Hash + Eq, V>(RefCell<FxHashMap<K, Box<V>>>);
 
 impl<K: Hash + Eq, V> MonoHashMap<K, V> {
-    pub fn values<T>(&self, f: impl FnOnce(&mut dyn Iterator<Item=&V>) -> T) -> T {
-        f(&mut self.0.borrow().values().map(|v| &**v))
+    /// This function exists for priroda to be able to iterate over all evaluator memory
+    ///
+    /// The memory of constants does not show up in this list.
```

Comment:
That last comment is somewhat odd here, because the hashmap isn't even a-priori specific to memory.

Reply:
true, but we don't use it anywhere else ;)

Reply:
I still feel like it'll create more confusion than it solves, at this place.  Might be better to say this wherever priroda uses this, to remind the reader of *that* code?